modules/gis/test/data/geojson_no_properties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "geometry": {
+        "type": "Point",
+        "coordinates": [100, 0]
+      }
+    },
+    {
+      "geometry": {
+        "type": "LineString",
+        "coordinates": [[100, 0], [101, 1]]
+      }
+    },
+    {
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [[[100, 0], [101, 0], [101, 10], [100, 10], [100, 0]]]
+      }
+    }
+  ]
+}
```

Comment:
This isn't a valid GeoJSON. A `FeatureCollection` must hold an array of `Feature` objects. I don't believe it can hold `Geometry` objects directly.